sdk-docs/project-structure.md

```diff
@@ -0,0 +1,93 @@
+    - `package.json`: node dependencies for only this app.
+    - `project.json`: holds project-specific settings and build configurations and is used by the Nx CLI to understand how to build, test, and serve your application or library.
+    Beyond what our guides tell you to do, you likely won't have to look too much into this file.
+    - `webpack.config.ts`: configuration file used by Webpack, a popular open-source JavaScript module bundler. Under normal circumstances you will never have to modify this file.
+
+### libs
+The libs folder should contain all the extensions, tools and helper-functions that you might want to share between apps. When you generate your first extension, you will have a folder structure looking roughly like this:
+- `/[feature_name]`: We suggest a convention where you bundle all extensions related to an overarching "feature" under a subcategory named after the feature. You don't have to do this and can omit creating this parent directory.
+    - `/[extension_name]`:
+        - `/src`: This folder should house all your code. Create directories and sub-directories for your tools, helper functions etc. here.
+            - `/generated`: This folder is automatically generated when you follow our graphql guide.
+                - `graphql-api.tsx`: It houses auto-generated hooks (functions) that call the graphql api for you.
+            - `/queries`: You should create this folder manually and put all your custom graphql queries in it. The hooks above are generated based on the queries you store in this folder
+                - `[query_name].graphql`: This is one of your custom graphql queries.
+            - `app.spec.tsx`: Unit tests for the App component.
+            - `app.tsx`: The entry-point of your custom code logic. Start coding here!
+            - `index.tsx`: The entry-point of your extension. It will initialize the App component above and you shouldn't do any changes here. Your point of interest is app.tsx.
+
+
+
+
+## Configuration Files
+### iris-app-manifest.ts
+Possibly the most important configuration file that you need to understand.
+It configures the following crucial settings for your app:
+-  `specVersion`: The version of the app manifest file. Should not be changed manually. 
+- `moduleFederationName`: Name of your app including workspace name, automatically parsed from your app-specific package.json.
+- `dependencies`: A full list of runtime dependencies for the app. Automatically parsed from the package.json in root. The list will be security scanned.
+- `devDependencies`: Same as above but for dev-dependencies. These are also security scanned.
+- `validDomains`: A list of any external domains that the app needs to communicate with. The list is audited in the approval step of the app and every external API call must be listed here to get through the verification process.
```

Comment:
Perhaps mention that any external URLs not listed here will not be possible to access. This is also the case for local development.

It's not only for verification, but there are mechanisms in place to block any URLs not included here. 😊